src/core/pattern.js

```diff
@@ -96,6 +96,7 @@ Shadings.RadialAxial = (function RadialAxialClosure() {
     var cs = dict.get('ColorSpace', 'CS');
     cs = ColorSpace.parse(cs, xref, res, pdfFunctionFactory);
     this.cs = cs;
+    this.bbox = dict.getArray('BBox');
```

Comment:
Would it make sense to normalize the bounding box here (as is done elsewhere in this file), i.e. `Util.normalizeRect(dict.getArray('BBox'));`, to prevent future issues?

Reply:
I agree since we have seen breakage because of that before (see #8168 and #10162).